_rules/SC4-1-2-aria-hidden-focus.md

```diff
@@ -0,0 +1,130 @@
+---
+name: aria-hidden with focusable content
+description: |
+  This rule checks `aria-hidden` elements do not contain focusable elements
+
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any element with a `aria-hidden="true"` attribute.
+
+**Note**: Using `aria-hidden="false"` on a decendent of an element with `aria-hidden="true"` **does not** expose that element. `aria-hidden="true"` hides everything inside of it from assistive technologies.
```

Comment:
editorial: hides itself and its children element from ..... rather than everything inside?